### ISSUE TITLE
[BUG]Console Error

### ISSUE THREAD

**Issue by bloggy** [ShaneBeee/SkBee#20]
Error:

[19:54:11] [Server thread/INFO]: KatzTatz issued server command: /ec
[19:54:16] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to SkBee v1.3.2
java.lang.NullPointerException: null

This server is running Tuinity version git-Tuinity-"55e85bb" (MC: 1.15.2) (Implementing API version 1.15.2-R0.1-SNAPSHOT)
Loading SkBee v1.3.2

Command is an essentials alias, nothing to do with skript

**Comment by ShaneBeee** [comment]
Im going to need the rest of that error, everything after this line:
```
java.lang.NullPointerException: null
```
is the important part.

**Comment by Govindass** [comment]
I am having the same issue, sometimes there is something after the NPE, sometimes it is just the NPE
example:
```
>.... [15:22:39 ERROR]: Could not pass event PlayerInteractEvent to SkBee v1.3.3
java.lang.NullPointerException: null
>.... [15:23:06 ERROR]: Could not pass event PlayerInteractEvent to SkBee v1.3.3
java.lang.NullPointerException: null
>.... [15:23:39 ERROR]: Could not pass event PlayerInteractEvent to SkBee v1.3.3
java.lang.NullPointerException: null
```
and here's a long one:
```
[13:44:52] [Server thread/ERROR]: Could not pass event PlayerInteractEvent to SkBee v1.3.3
java.lang.NullPointerException: null
	at tk.shanebee.bee.api.virtualfurnace.api.FurnaceManager.getFurnaceFromItemStack(FurnaceManager.java:258) ~[?:?]
	at tk.shanebee.bee.api.virtualfurnace.api.FurnaceListener.onClickFurnace(FurnaceListener.java:44) ~[?:?]
	at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at org.bukkit.craftbukkit.v1_15_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:471) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.PlayerInteractManager.a(PlayerInteractManager.java:483) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.PlayerConnection.a(PlayerConnection.java:1406) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at xuan.cat.XuanCatAPI.a.b.c.a.g.a(Unknown Source) ~[?:?]
	at xuan.cat.XuanCatAPI.a.b.c.a.g.a(Unknown Source) ~[?:?]
	at xuan.cat.XuanCatAPI.a.b.c.a.g.a(Unknown Source) ~[?:?]
	at net.minecraft.server.v1_15_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:27) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.PacketPlayInUseItem.a(PacketPlayInUseItem.java:5) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.PlayerConnectionUtils.lambda$ensureMainThread$0(PlayerConnectionUtils.java:43) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.TickTask.run(SourceFile:18) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.MinecraftServer.ba(MinecraftServer.java:1151) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.MinecraftServer.executeNext(MinecraftServer.java:1144) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.MinecraftServer.sleepForTick(MinecraftServer.java:1120) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:963) ~[patched_1.15.2.jar:git-Tuinity-"64e46c6"]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by ShaneBeee** [comment]
Hmm, ok thanks for letting me know, Ill look into this.

**Comment by bloggy** [comment]
I do not have more than this line.

I just discovered another NPE in console issue:

https://pastebin.com/QnYnZCiW

**Comment by bloggy** [comment]
Same error again, no fix yet?

https://pastebin.com/ng5C6XL9

**Comment by ShaneBeee** [comment]
This should hopefully be fixed in the latest update [**1.4.1**](https://github.com/ShaneBeee/SkBee/releases/tag/1.4.1)
Since you didn't post any code or anything to replicate I'm not able to test the issue, but I put a null check in so hopefully it solves this.
